textstat/textstat.py

```diff
@@ -177,6 +177,17 @@ def lexicon_count(self, text, removepunct=True):
         count = len(text.split())
         return count
 
+    @lru_cache(maxsize=128)
+    def miniword_count(self, text, removepunct=True):
+        """
+        Function to return total miniword (Common words with three letters
+        or less) counts in a text
+        """
+        if removepunct:
+            text = self.remove_punctuation(text)
+        count = len([word for word in text.split() if len(word) < 4])
```

Comment:
I see. My immediate inclination is to remove `removepunct` from the function and always make it run `self.remove_punctuation(text)` as that seems to reliably get a good count of words with `.split` and I'm not sure when we would want to call `textstat.miniword_count(text, False)`
However, I notice that `textstat.lexicon_count` also has the `removepunct` argument, so I'm wondering if there is some reason we would want to keep that argument for collecting word counts?

Reply:
I've just had a look through, and `lexicon_count` does only do a `.split()`. I thought it did more.

If we don't remove punctuation we can have (as an example from the tests) `"..."`, and `"&"` as words.

There is a `long_word_count` already defined, so it might be best just to stick with what this is already doing.

```python
    def long_word_count(self, text):
        ''' counts words with more than 6 characters '''
        word_list = self.remove_punctuation(text).split()
        return len([w for w in word_list if len(w) > 6])
```